modules/reporting/app/controllers/cost_reports_controller.rb

```diff
@@ -566,4 +592,8 @@ def find_optional_report(query = "1=0")
     end
   rescue ActiveRecord::RecordNotFound
   end
+
+  def session_name
+    report_engine.name.underscore.to_sym
```

Comment:
```suggestion
    @session_name ||= report_engine.name.underscore.to_sym
```

just a micro optimization as we seem to call this a lot